<launch>
  <node name="image_viewer" pkg="rab_bringup" type="image_viewer" output="screen">
    <param name="odom_topic" value="/amcl_pose" />
  <!-- Set image near main gate -->
    <param name="first_path" value="$(find rab_bringup)/picture/nobori.jpg" />
    <param name="first_x" value="228.436" />
    <param name="first_y" value="174.79" />
  <!-- Set image to guide -->
    <param name="second_path" value="$(find rab_bringup)/picture/lab_caution.jpg" />
    <param name="second_x" value="228.436" />
    <param name="second_y" value="174.79" />
  <!-- Set image to 4th to main gate -->
    <param name="default_path" value="$(find rab_bringup)/picture/sample.JPG" />
  </node>
</launch>
